src/ticketing/utils.ts

```diff
@@ -18,7 +18,7 @@ import {getLastUsedAccess} from '@atb/fare-contracts/utils';
 export function isCarnetTravelRight(
   travelRight: TravelRight | undefined,
 ): travelRight is CarnetTravelRight {
-  return travelRight?.type === 'CarnetTicket';
+  return !!travelRight && 'maximumNumberOfAccesses' in travelRight;
```

Comment:
@gorandalum You've talked a lot about not using the `type` field for logic in the app: Do you think this is this an ok replacement?

Reply:
Yes, that's great, directly from the source :+1: